_pages/welcome-to-TTS/classes/gsa-internal-tools.md

```diff
@@ -2,66 +2,31 @@
-      <td>To reserve your own conference call line</td>
-      <td>ENT username and password</td>
-    </tr>
-    <tr>
-      <td><a href="{{site.baseurl}}/olu/">Online University</a></td>
-      <td>To take mandatory trainings for federal employees</td>
-      <td>ENT username and password</td>
-    </tr>
-  </tbody>
-</table>
+Most of GSA's internal tools are accessible via our Agency's intranet [Insite](http://insite.gsa.gov). You must be on the VPN to connect if you work remotely. [Instructions for connecting are here]({{site.baseurl}}/networks/). Here's an overview of what you can access (you can click the names in this table to scroll down the page):
+
+| Name                                             | What we use it for                                                                                                                                         | Login system                           |
+| ------------------------------------------------ | ---------------------------------------------------------------------------------------------------------------------------------------------------------- | -------------------------------------- |
+| [BookIT!](#bookit)                               | To reserve a desk or meeting room in GSA buildings                                                                                                         | ENT username and password              |
+| [Concur](https://travel.gsa.gov)                 | To book travel. Learn more in [Travel 101]({{site.baseurl}}/travel-101/).                                                                                  | ENT username and password              |
+| [Employee Express](#employee-express)            | To view your paystub, change your withholdings, or change your address                                                                                     | Employee Express username and password |
+| [HR Links](#hr-links)                            | To request/document leave and access your personnel file online                                                                                            | ENT on GSA Wireless or VPN             |
+| [IT Service Desk](#it-service-desk)              | General IT questions about Employee Express and other passwords (Note: They may not support requests for Macs, but will support requests for GSA websites) | ENT username and password              |
+| [Meeting Space](#meeting-space)                  | To reserve your own conference call line                                                                                                                   | ENT username and password              |
+| [Online University (OLU)]({{site.baseurl}}/olu/) | To take mandatory trainings for federal employees                                                                                                          | ENT username and password              |
```

Comment:
Can't decide if I like the Markdown or the HTML table better 😕

Reply:
Semantically, I think a DL is more appropriate than a table, FWIW

Reply:
Given the simplicity and consistency of the clauses, a UL might be an option, too. Something along the lines of...

### You can access
* [link](HRlinks) to do hrlinks stuff (and frustrate yourself)
* [link](Bookit) to reserve a room

Just a thought.

Reply:
No strong feelings. Care to send as a follow-up pull request?